generators/aws-containers/prompts.js

```diff
@@ -281,8 +278,7 @@ function promptPerformance(config, awsConfig = { performance: 'low' }) {
     ];
 
     return this.prompt(prompts).then(props => {
-        const performance = props.performance;
-        return performance;
+        return props.performance;
```

Comment:
I don't see a reason for 
```
.then(props => {
         return props.performance;
})
```

`return this.prompt(prompts)`
should be enough.

Reply:
I was merely removing the declaration of a redundant const.

I can change that to your proposed suggestion, however wouldn't that change the semantics of the existing logic?

Reply:
This promise isn't used anywhere.
https://github.com/jhipster/generator-jhipster/search?q=promptPerformance&unscoped_q=promptPerformance

Reply:
Ah! Makes sense now. 👍

Reply:
Sorry for asking again but, wouldn't it break https://github.com/jhipster/generator-jhipster/blob/d3cc5f6b8e021422bbab6d8151172ececb0eaf2d/generators/aws-containers/prompts.js#L231
`.then(performance => {`

Reply:
You are right.